features/vesting/selectVesting.tsx

```diff
@@ -0,0 +1,56 @@
+import { FC, ReactNode, useCallback, useEffect, useMemo } from 'react';
+import { SelectIcon, Option, SelectIconProps } from '@lidofinance/lido-ui';
+import { AddressBadge } from 'shared/ui';
+import { useVestingsContext } from './vestingsProvider';
+
+type SelectVestingProps = Omit<SelectIconProps, 'icon'> & {
+  error?: string | ReactNode;
+};
+
+export const SelectVesting: FC<SelectVestingProps> = ({
+  name,
+  onChange,
+  error,
+  ...rest
+}) => {
+  const { vestings, currentVesting, setCurrentVesting } = useVestingsContext();
+  const orderedVestings = useMemo(
+    () => vestings?.slice()?.reverse(),
+    [vestings],
+  );
+
+  useEffect(() => {
+    onChange?.(currentVesting);
+  }, [onChange, currentVesting]);
```

Comment:
looks weird

Reply:
This was made, so `selectVesting` can behave more or less the same way to inputs, it's not important right now, since this select will be heavily refactored in upcoming UI redesign.